source/includes/steps-install-bi-connector-macos.yaml

```
@@ -6,6 +6,72 @@ source:
   ref: download-bi
 ---
 stepnum: 2
+title: Verify the integrity of the downloaded package.
+ref: verify-bi-mac-package
+level: 4
+content: |
+  The MongoDB release team digitally signs all software packages to
+  certify that a particular MongoDB package is a valid and unaltered
+  MongoDB release. MongoDB signs each release branch with a different
+  PGP key in ``.asc`` format.  
+
+  a. Run the following command to downloaded the ``.sig`` file. 
+       
+     .. code-block:: sh 
+            
+        curl -LO https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-osx-{x86_64|arm64}-{version}.zip.sig
```

Here is are example links : 
- https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-osx-arm64-macos11-v2.14.14.zip.sig
- https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-osx-x86_64-v2.14.14.zip.sig
```suggestion
        curl -LO https://info-mongodb-com.s3.amazonaws.com/mongodb-bi/v2/mongodb-bi-osx-{x86_64|arm64-macos{11|13}}-{version}.zip.sig
```